src/components/ThreadEnvelope.vue

```diff
@@ -21,7 +21,12 @@
 
 <template>
 	<div>
-		<div class="icon-mail">
+		<div class="avatar">
```

Comment:
This is no longer the avatar but avatar + sender/subject, but since there is no styles defined anyways you can probably just drop the class

Reply:
leftovers :) thank you